util/misc.go

```diff
@@ -189,6 +189,11 @@ func IsMemDB(dbLowerName string) bool {
 	return false
 }
 
+// IsSysDB checks whether dbLowerName is system database.
+func IsSysDB(dbLowerName string) bool {
```

Comment:
So is it allowed to set replicas for `information_schema`?

Reply:
Hard to say, from the requirements, MySQL is the only one mentioned.